cli/template/extras/src/app/page/with-lucia-trpc-tw.tsx

```diff
@@ -0,0 +1,87 @@
+import Link from "next/link";
+
+import { CreatePost } from "~/app/_components/create-post";
+import { LogOutButton } from "~/app/_components/logout-button";
+import { getPageSession } from "~/server/auth";
```

Comment:
Maybe use a same-named export for both NextAuth and lucia so we can use the same frontend stuff for both?